modules/installation-restricted-network-samples.adoc

```diff
@@ -29,6 +32,9 @@ ifdef::samplesoperatoraltreg[]
 Operator.
 endif::[]
 
+Setting the `registryoverride` field in the Sample Operator configuration file to link:registry.redhat.io[registry.redhat.io] is redundant because it is already directed to link:registry.redhat.io[registry.redhat.io] for eveything but Jenkins images and imagestreams. It also breaks the installation payload for Jenkins imagestreams. 
```

Comment:
Coincidently @bmcelvee another BZ along these lines appears to have come in today.

I'm now now leaning toward explicitly preventing the override of the registry for the jenkins images to 

`"docker.io", "registry.redhat.io", "registry.access.redhat.com", "quay.io", "registry.svc.ci.openshift.org"`

As such, leave this text as is, but add the new sentence: `As such, the sample operator prevents the use of these registries for the jenkins imagestreams:  "docker.io", "registry.redhat.io", "registry.access.redhat.com", "quay.io"`

We'll leave out our internal CI registry.